mod-linked-data/src/main/resources/karate-config.js

```diff
@@ -97,13 +97,6 @@ function fn() {
     }
     config.prototypeTenant = '${prototypeTenant}';
     karate.configure('ssl',true);
-  } else if (env == 'ecs') {
```

Comment:
Removing these to fix karate test failure in jenkins:

java.lang.Exception: Caught exception: groovy.lang.GroovyRuntimeException: Failed to parse template script (your template may contain an error or be trying to use expressions not currently supported): startup failed:
SimpleTemplateScript981.groovy: 105: illegal string body character after dollar sign;
   solution: either escape a literal dollar sign "\$5" or bracket the value expression "${5}" @ line 105, column 29.
         password: 'LinkedData$01'